CONTRIBUTING.md

```diff
@@ -38,13 +38,13 @@ The kind of code we can accept depends on the life cycle for the release you're
 
 At the end of the PR description, which is autofilled with the commit message if there is only one commit, add the phrase, "Fixes scala/bug#NNNN", where `https://github.com/scala/bug/issues/NNNN` tracks the bug you're fixing. Github will turn your bug number into a link.
 
-We also recommend naming your branch after the ticket number.
+We also recommends naming your branch after the ticket number.
 
-Please make sure the ticket's milestone corresponds to the upcoming milestone for the branch your PR targets. The CI automation will automatically assign the milestone after you open the PR.
+Please make sure the ticket's milestone corresponds to the upcoming milestone for the branch of your PR targets. The CI automation will automatically assign the milestone after you open the PR.
 
 #### Enhancement or New Feature
 
-For longer-running development, likely required for this category of code contributions, we suggest you include `topic/` or `wip/` in your branch name, to indicate that this is work in progress, and that others should be prepared to rebase if they branch off your branch.
+For longer-running development, likely required for this category of code contributions, we suggest you include `topic/` or `wip/` in your branch name, to indicate that this is work in progress and that others should be prepared to rebase if they branch off your branch.
```

Comment:
I'd argue that `and that others...` is also part of the subordinate phrase `that this is work in progress` and therefore the comma is superfluous.

That said, you can't change the source branch after a PR is made (right?) so maybe the advice should be to use GitHub's draft-PR feature.

Reply:
The comma before "to indicate" is the incorrect punctuation in the sentence, IMO.

Reply:
🎶 Comma, comma, comma chameleon... 🎶